aspnetcore/blazor/components.md

```diff
@@ -101,6 +104,15 @@ Rendering server components from a static HTML page isn't supported.
 
 For more information on how components are rendered, component state, and the `Component` Tag Helper, see <xref:blazor/hosting-models>.
 
+## Tag Helpers aren't used in components
```

Comment:
This text makes some claims about tag helpers and components that I think are unnecessary. I think all we want to say here is:

```
[Tag Helpers](xref:mvc/views/tag-helpers/intro) are not supported in Razor components (.razor files). To provide Tag Helper-like functionality in Blazor, create a component with the same functionality as the Tag Helper and use the component instead.
```